src/ucp/wireup/select.c

```diff
@@ -1093,8 +1094,14 @@ static ucs_status_t ucp_wireup_add_rma_bw_lanes(ucp_ep_h ep,
     bw_info.max_lanes         = ep->worker->context->config.ext.max_rndv_lanes;
     bw_info.usage             = UCP_WIREUP_LANE_USAGE_RMA_BW;
 
-    return ucp_wireup_add_bw_lanes(ep, address_count, address_list, &bw_info, 0,
-                                   lane_descs, num_lanes_p);
+    for (mem_type = 0; (mem_type < UCT_MD_MEM_TYPE_LAST) &&
+         (ep->worker->context->mem_type_tls[mem_type] != 0); mem_type++) {
```

Comment:
why is (ep->worker->context->mem_type_tls[mem_type] != 0) a stop condition? seems like each mem type should be checked independently..

Reply:
my bad.   Actually I wanted to skip ucp_wireup_add_bw_lanes if  no TLs for mem_type.